Species/The Skrell.md

```diff
@@ -161,6 +183,20 @@ Teachers and Professors have the same purpose, having the ability to vote the la
 
 Directors and other leading figures of the Academies often being Qerr-Koal, they do not take part in the legislative part of the Skrellian political life.
 
+### Economy
+
+For the Skrell, money is an ephemeral thing. Skrellian currencies are printed with a temporary filigree, made from the spores of a very specific luminescent mushroom on Qerr’balak. The luminescence fades completely after two years, after which a Skrellian bill is no longer usable. Each filigree includes the date of the bill’s printing, allowing the verification of the bill’s authenticity by comparing the date and the expected luminosity of the spores.
+
+This means Skrell cannot hoard money, and are instead incentivized to spend it, be it by buying goods and services, or investing it in various endeavors. If a Skrell wishes to preserve money for future projects, or simply for financial safety, their only recourse is to loan that money, without interest, to the City-State, for a set duration decided at the start of the loan. When the government gives back the money, the bills are freshly printed for a new 2-year long period of use. 
+
+That way, the Skrellian City-States are consistently supplied with their citizens’ money, negating entirely the need for taxes or banks in their society. Since money is constantly printed and destroyed, inflation is a very minor phenomenon, and the gap of wealth between the various casts, while still existent, is far less consequent than in humans societies. 
```

Comment:
Probably, yeah. Call it a translation error, as "consequent" means "consequential" in French. Fixing it.